Gemfile

```diff
@@ -38,6 +38,7 @@ group :test do
 end
 
 group :assets do
+  gem 'sass', '3.2.19'
```

Comment:
I think rather than adding `sass` as an explicit dependency, it'd be better to update the `sass` dependency in `Gemfile.lock`. You can do this using `bundle update sass`.